code/game/gamemodes/miniantags/abduction/abduction_outfits.dm

```diff
@@ -1,6 +1,5 @@
 /datum/outfit/abductor
 	name = "Abductor Basic"
-	uniform = /obj/item/clothing/under/color/grey //they're greys gettit
```

Comment:
Make this `/obj/item/clothing/under/abductor` and remove the change from the agent.

Reply:
Done, @FlattestGuitar